### DISCUSSION TITLE
EMQX Cloud

### DISCUSSION THREAD

**Discussion by aze2201** [emqx/emqx#10689] [General]
Last time I checked EMQX Enterpise in Azure VM and able to get necessary features what I need.
Now I am checking our EMQX Cloud (Dedicated plan) and cannot find some features. Can anybody help me get below features whether are exist or not?

1. Authenticate with self-seigned certificate only. It says I have to Add username to `ACL `which accepts `Username `and `Password`. And I don't want use `username/password.`
2. How to bind certificate `Username `to common name `CN`?
3. How can I make `ACL `to accept if `Username = CN = Topicname`
Like 
```
%% EMXQ Enterpise Azure VM
{allow, all, pubsub, ["%u/#"]}.
{deny, all}.  
```

I added `server.key|.crt` and `ca.crt`, but could not authenticate to emqx cloud because of no username exist.

**Comment by id** [comment]
Hi @aze2201.

Thank you for choosing EMQX Cloud and for providing us with your feedback and inquiries. To ensure that we can provide efficient and professional technical support, we kindly recommend submitting all EMQX Cloud-、related questions through our console. This will allow our support team to promptly respond and provide solutions.

By submitting a ticket through the console, we can better track and document your issues and match them with the appropriate resources and experts. This enables us to better understand the challenges you are facing and offer more targeted solutions, ensuring that your experience is maximized.
Furthermore, submitting a ticket through the console ensures that your inquiries receive timely attention without getting lost in discussions within public communities. We value your time and strive to provide you with prompt and personalized assistance through our dedicated technical support channel.

Please proceed to our console and click on the relevant support option to submit your ticket. Our support team will respond as soon as possible and provide you with a satisfactory resolution. Thank you for your understanding and cooperation!

Wishing you a pleasant experience with EMQX Cloud!

Best regards,
The EMQX Cloud Support Team
